operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

```diff
@@ -393,7 +394,7 @@ Scenarios where this procedure is applicable:
     deployment "cray-reds" successfully rolled out
     ```
 
-1. **Skip if adding `ncn-m001`:** Wait for the NCN BMC to get discovered by HSM:
+1. (`ncn-m#`) **Skip if adding `ncn-m001`:** Wait for the NCN BMC to get discovered by HSM:
     > If the BMC of `ncn-m001` is connected to the site network, then we will be unable to discover the BMC, because it is not connected via the HMN network.
```

Comment:
```suggestion
    > If the BMC of `ncn-m001` is connected to the site network, then the BMC will not be discovered, because it is not connected via the HMN network.
```